<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infovenci</class>
 <widget class="QDialog" name="infovenci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Información sobre vencimientos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fecha inicial:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="inicial_dateEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fecha final:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="final_dateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Plazo máximo de pago (días):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="diaslineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Restringir cuentas a evaluar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cuentaslineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="consulta_tableWidget">
     <property name="rowCount">
      <number>5</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Concepto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Valor</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refrescarpushButton">
       <property name="text">
        <string>&amp;Refrescar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/rayo.png</normaloff>:/iconos/graf/rayo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="informepushButton">
       <property name="text">
        <string>&amp;Imprimir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/printer.png</normaloff>:/iconos/graf/printer.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ed_latex_pushButton">
       <property name="text">
        <string>&amp;Latex</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/tex.png</normaloff>:/iconos/graf/tex.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inicial_dateEdit</tabstop>
  <tabstop>final_dateEdit</tabstop>
  <tabstop>diaslineEdit</tabstop>
  <tabstop>cuentaslineEdit</tabstop>
  <tabstop>consulta_tableWidget</tabstop>
  <tabstop>refrescarpushButton</tabstop>
  <tabstop>informepushButton</tabstop>
  <tabstop>ed_latex_pushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>infovenci</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
